feature/isis/feature.textproto

```diff
@@ -141,6 +141,9 @@ config_path {
 telemetry_path {
   path: "/network-instances/network-instance/protocols/protocol/isis/interfaces/interface/levels/level/afi-safi/af/state/metric"
 }
+config_path {
```

Comment:
nit: from order perspective, can we put the config path before the telemetry path blob? Just to be consistent.

Reply:
Done!